go/controller/cluster.go

```diff
@@ -95,6 +95,26 @@ func getPodsTotalRequestsAndLimits(podList *v1.PodList) (reqs v1.ResourceList, l
 	return
 }
 
+func syncNodesIdleResource(podList *v1.PodList, nodesCPUIdleMilli map[string]int64, nodesMemoryIdleMega map[string]int64) (err error) {
+	for _, pod := range podList.Items {
+		nodeName := pod.Spec.NodeName
+		// if nodeName is empty, the pod is not assigned.
```

Comment:
This comment seems not needed, the below code is clear enough.

Reply:
Done.